OrbitGl/TracepointTrack.cpp

```diff
@@ -81,4 +119,37 @@ float TracepointTrack::GetHeight() const {
 
 bool TracepointTrack::HasTracepoints() const {
   return !GOrbitApp->GetCaptureData().GetTracepointsOfThread(thread_id_).empty();
+}
+
+void TracepointTrack::OnPick(int x, int y) {
+  Vec2& mouse_pos = mouse_pos_[0];
+  canvas_->ScreenToWorld(x, y, mouse_pos[0], mouse_pos[1]);
+  mouse_pos_[1] = mouse_pos_[0];
+  picked_ = true;
+}
+
+void TracepointTrack::OnRelease() { picked_ = false; }
+
+std::string TracepointTrack::GetSampleTooltip(PickingId id) const {
+  static const std::string unknown_return_text = "Linux tracepoint event information missing";
+
+  auto user_data = time_graph_->GetBatcher().GetUserData(id);
+  if (!user_data || !user_data->custom_data_) {
+    return unknown_return_text;
```

Comment:
Can this ever happen? Maybe write something in the error log.

Reply:
It should not happen. I will do a CHECK on it. With this occasion, we will also see if everything works as expected. Thank you!